ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/zscaler_endpoints.schema.yml

```diff
@@ -11,13 +11,8 @@ keys:
     description: |-
       PREVIEW: These keys are in preview mode.
 
-      Special data model used for WAN internet-exit integration with Zscaler.
-      The model is supposed to be autofilled per-device using the `arista.avd.cv_zscaler_endpoints` Ansible lookup plugin.
-      Example:
-      ```yaml
-      zscaler_endpoints: "{{ lookup('arista.avd.cv_zscaler_endpoints', cv_server="<optional servername>", cv_token="<optional token>") }}"
-      ```
-      The cv_server and cv_token options can be set as inventory variables instead of given as arguments. (Then they will also be used for cv_deploy)
+      Special data model used for testing the WAN internet-exit integration with Zscaler.
+      The model is supposed to be autofilled per-device by `eos_designs`, but when CloudVision is not available, it is possible to provide the settings manually.
```

Comment:
Actually if the key is set it will take over and CV will not be called as per the code so we should probably write it like this in the doc?